packages/card/src/CardTitle.js

```diff
@@ -28,6 +28,8 @@ type Props = {|
   meta?: string,
   /** Title of the card */
   children: MnrlReactNode,
+  /** HTML class applied to the root element */
+  className?: string,
```

Comment:
I wasn’t sure if this was the right approach, or if we should simply pass along all props like we do for other components. I didn’t do that here, because CardTitle is only to be used inside of Card.